File: 52a/main.jl
print("length: ")
input1 = readline()
length = parse(Int64, input1)

print("width: ")
input2 = readline()
width = parse(Int64, input2)

area = width * length
perimeter = (width * 2) + (length * 2)

# length: 143
# width: 82
# julia> show(area)
# 11726
# julia> show(perimeter)
# 450

File: 52a/main.kt
import java.util.Scanner
fun main(args: Array<String>) {
    val reader = Scanner(System.`in`)
    print("length: ")
    var length:Double = reader.nextDouble()

    print("width: ")
    var width:Double = reader.nextDouble()

    var area: Double
    area = length * width

    var perimeter: Double
    perimeter = (length * 2) + (width * 2)
    println("area: $area")
    println("perimeter: $perimeter")
}

/*
length: 143
width: 82
area: 11726.0
perimeter: 450.0

Process finished with exit code 0
 */